mtp_api/apps/core/filters.py

```diff
@@ -14,6 +13,7 @@
 import django_filters
 import django_filters.fields
 import django_filters.utils
+import six
```

Comment:
line 140 can be turned into this actually: `get_all_format_lazy = lazy(get_all_format, list)`
i can't see why tuple or str needs to be in there

Reply:
I don't know much about that so I'll just replace `six.text_type` with `str` in case there is a reason 🤷

Reply:
`get_all_format` returns a list so i think the other options aren't necessary